py3-gast.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: py3-gast
+  version: 0.5.4
+  epoch: 0
+  description: Python AST that abstracts the underlying Python version
+  copyright:
+    - license: BSD 3-Clause
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/serge-sans-paille/gast
+      tag: ${{package.version}}
+      expected-commit: 1f872364a9964bdb739c4f76d0be463104cc4757
+
+  - name: Python Build
+    runs: python setup.py build
+
+  - name: Python Install
+    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+# no versions returned from github graphql api
```

Comment:
I can see the tags matching the version [here](https://github.com/serge-sans-paille/gast/releases/tag/0.5.4) 

so we should be able to use
```
update:
  enabled: true
  github:
    identifier: serge-sans-paille/gast
    use-tag: true
```
which I validated using 

```
wolfictl check update  --override-version 0.0.1 py3-gast
```

Reply:
I didn't use `use-tag`, let me try that again with this option.

Reply:
thanks James!